packages/ui/src/molecules/ProductTitle/stories/ProductTitle.mdx

```diff
@@ -0,0 +1,20 @@
+import { Canvas, Props, Story, ArgsTable } from '@storybook/addon-docs'
+
+import ProductTitle from '../ProductTitle'
+
+# ProductTitle
+
+<Canvas>
+  <Story id="molecules-producttitle--product-title" />
+</Canvas>
+
+## Props
+
+<ArgsTable of={ ProductTitle } />
+
+## CSS Selectors
+
+```css
+[data-store-product-title] {
+}
```

Comment:
Missing the new `fs` format a few other selectors:
```suggestion
[data-fs-product-title] {
}

[data-fs-product-title-header] {
}

[data-fs-product-title-addendum] {
}
```

Reply:
Actually, I'm not sure about it, because on the `faststore/ui`, we are using the `data-store` pattern. Did we change it again? @filipewl @ArthurTriis1

Reply:
But if we change to `data-store` pattern here we will need to add new data-attr when importing on starters so the theming can work 🤔

Reply:
We will change all attributes in faststore to use the data-fs  attribute pattern, for we can use the theming.